src/vm/eetwain.cpp

```diff
@@ -562,7 +562,8 @@ FrameType   GetHandlerFrameInfo(hdrInfo   * info,
     // The slots grow towards lower address on the stack and is terminted by a NULL entry.  
     // Since each subsequent slot contains the SP of a more nested EH clause, the contents of the slots are
     // expected to be in decreasing order.
-    size_t lvl;
+    size_t lvl = 0;
```

Comment:
Yes, the shadow SP slots that this change is ifdefing out should not be needed for WIN64EXCEPTIONS.